network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_index_callout_box.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags static i18n %}
+
+<div class="tw-h-full tw-flex tw-flex-col tw-justify-between tw-p-[36px] tw-border tw-border-black tw-shadow-[10px_10px_rgba(0,0,0)]">
+  <div id="callout-box-body" class="tw-mb-[4rem]">
+        <div class="tw-flex">
+          {% if value.related_topics %}
+            <div class="tw-mb-3">         
```

Comment:
**Question** Since we have that margin on all children can we move that to the parent element?